### PR TITLE
Proofreading james coglan captions

### PR DESCRIPTION
**Pull request by Harxy**

### CODE REVIEW

File: 11/chapter-12.srt [hunk 1]
```diff
@@ -54,7 +54,7 @@ to do that, but we'll get to that in a second.
 
 13
 00:01:20,620 --> 00:01:25,040
-"undefined method `>' for nil:NilClass
+"undefined method `>' for nil:NilClass"
```
Comment (nelstrom) [inline 11/chapter-12.srt:57]: 👀

File: 11/chapter-12.srt [hunk 2]
```diff
@@ -564,28 +564,28 @@ One in target spec and one in the integration tests.
...
+regeneration_time and got echo high.
 
 124
 00:12:16,020 --> 00:12:23,074
 That's because the end of our build self method...
 
 125
 00:12:23,142 --> 00:12:29,234
-The last thing in that is a inaud instructions,
+The last thing in that is an each over the instructions,
```
Comment (nelstrom) [inline 11/chapter-12.srt:575]: Good catch, thanks!

**Comment by nelstrom** [comment]
Great job @Harxy, thanks!

File: 11/chapter-12.srt (hunk 1 above)
Comment (Harxy) [reply 11/chapter-12.srt:57]: I just thought if you were going to start the quotes, they should be finished? Possibly not needed at all. Just a thought.

File: 11/chapter-12.srt [hunk 3]
```diff
@@ -332,8 +332,8 @@ the mtime of that file at that point.
 
 73
 00:07:31,051 --> 00:07:35,622
-But your talking about almost the end
-point of that file after it's been regenerated.
+But your talking about almost the mtime
```
Comment (Harxy) [inline 11/chapter-12.srt:335]: this should also be you're not your! I missed it... can you change it your end? Much easier.

File: 11/chapter-12.srt (hunk 1 above)
Comment (nelstrom) [reply 11/chapter-12.srt:57]: Oh yeah, I agree! Was that not obvious from my choice of emoji? 🔮

File: 11/chapter-12.srt (hunk 3 above)
Comment (nelstrom) [reply 11/chapter-12.srt:335]: Good catch! [Done](https://github.com/fromPeerToPeer/captions/pull/87/commits/9cf611b11f26bdd364d02fdda854527592e9d7c0)

**Pull request merged**